### DISCUSSION TITLE
Datetime picker for Tailwind

### DISCUSSION THREAD

**Discussion by cyggarro** [mdbootstrap/TW-Elements#1531] [Share ideas & request features]
This component should allow users to select a date and time using an interactive calendar and clock interface. 

The datetimepicker should include the ability to customize the date and time format, as well as the ability to set minimum and maximum selectable dates and times.

This should be a single component (instead of having to use Datepicker & Timepicker separately).

**Comment by filipkappa** [comment]
Thanks for submitting this idea!

It gathered so much traction that we have added a task for this feature: https://github.com/mdbootstrap/Tailwind-Elements/issues/1558

You can track our progress live here:
https://tailwind-elements.com/docs/standard/getting-started/changelog/

**Comment by filipkappa** [comment]
Good news! We introduced this feature in our latest release!
See here: #1899
